src/searcher.rs

```diff
@@ -98,12 +98,13 @@ impl Searcher {
     /// Returns `SearchResult` object.
     ///
     /// Raises a ValueError if there was an error with the search.
-    #[args(limit = 10, count = true)]
+    #[args(limit = 10, offset = 0, count = true)]
```

Comment:
The new argument needs to be documented.

Reply:
done!